collector.go

```diff
@@ -32,21 +32,21 @@ import (
 var (
 	metricMaps = map[string]map[string]ColumnMapping{
 		"databases": {
-			"name":                {LABEL, "N/A", 1, "N/A"},
-			"host":                {LABEL, "N/A", 1, "N/A"},
-			"port":                {LABEL, "N/A", 1, "N/A"},
-			"database":            {LABEL, "N/A", 1, "N/A"},
-			"force_user":          {LABEL, "N/A", 1, "N/A"},
-			"pool_size":           {GAUGE, "pool_size", 1, "Maximum number of server connections"},
-			"reserve_pool":        {GAUGE, "reserve_pool", 1, "Maximum number of additional connections for this database"},
-			"pool_mode":           {LABEL, "N/A", 1, "N/A"},
-			"max_connections":     {GAUGE, "max_connections", 1, "Maximum number of allowed connections for this database"},
-			"current_connections": {GAUGE, "current_connections", 1, "Current number of connections for this database"},
+			"name":                {LABEL, "name", 1, "Name of configured database entry."},
+			"host":                {LABEL, "host", 1, "Host pgbouncer connects to."},
+			"port":                {LABEL, "port", 1, "Port pgbouncer connects to."},
+			"database":            {LABEL, "database", 1, "Actual database name pgbouncer connects to."},
+			"force_user":          {LABEL, "force_user", 1, "When user is part of the connection string, the connection between pgbouncer and PostgreSQL is forced to the given user, whatever the client user."},
+			"pool_size":           {GAUGE, "pool_size_total", 1, "Maximum number of server connections"},
+			"reserve_pool":        {GAUGE, "reserve_pool_total", 1, "Maximum number of additional connections for this database"},
+			"pool_mode":           {LABEL, "pool_mode", 1, "The database's override pool_mode"},
+			"max_connections":     {GAUGE, "max_connections_total", 1, "Maximum number of allowed connections for this database"},
+			"current_connections": {GAUGE, "current_connections_total", 1, "Current number of connections for this database"},
```

Comment:
Why are you changing these existing metrics? From the documentation, these are clearly gauges. You are adding `_total` to gauges which is invalid.

Reply:
fixed